pkg/build/oci/oci.go

```diff
@@ -119,14 +129,50 @@ func buildImageFromLayer(layerTarGZ string, ic types.ImageConfiguration, created
 
 	v1Image, err = mutate.ConfigFile(v1Image, cfg)
 	if err != nil {
-		return empty.Image, fmt.Errorf("unable to update OCI config file: %w", err)
+		return nil, fmt.Errorf("unable to update OCI config file: %w", err)
 	}
 
-	return v1Image, nil
+	si := signed.Image(v1Image)
+
+	// Attach the SBOM, e.g.
+	if len(sbomFormats) > 0 {
+		var mt ggcrtypes.MediaType
+		var path string
+		switch sbomFormats[0] {
+		case "spdx":
+			mt = ctypes.SPDXMediaType
```

Comment:
I think this media type is intended for non-JSON sboms (like `ko` emits), and `apko` emits JSON sboms, so we should sort out the most appropriate media type to use here.

Reply:
cc @dlorenc

Reply:
I see cosign now has: `SPDXJSONMediaType`, I'll update to that.